### PR TITLE
Create Tech Radar

### LINKED ISSUE #5: Write an Endorsed Technologies Reference and Skill Mapping
**As an RTS developer** I need a list of endorsed technologies and the individuals possessing the required expertise associated with each of them.

The term Technology encompasses coding languages, frameworks, SDKs, and other tools essential for the development and implementation of projects.

## Acceptance Criteria

- List of technologies that are used today in the RTS ecosystem.
- Define the level of support of each technology by platform, e.g. : `No-go`, `Tolerated`, `Recommended`  (see [Technology Rader](https://www.thoughtworks.com/radar))
- List the individuals possessing the required expertise for each technology.

### PR DESCRIPTION
**Pull request by MGaetan89**
This PR aims at providing an overview of the technologies used in SRGSSR, as explained in #5.
It provides a workflow that runs every day, to get the list of primary languages and frameworks used inside the [SRGSSR GitHub organisation](https://github.com/SRGSSR).

## Done in this PR

- list the primary languages used in the public repositories of the SRGSS organisation
- list all the dependencies used by these repositories
- let the workflow create a PR to propose the changes

Before merging, the following commit should be reverted: https://github.com/SRGSSR/guilde-plateformes-propres/pull/11/commits/cc3c2cb45956fbc2b7e32a146a6095ca520b9137

> [!NOTE]
> The Markdown files in the `public/techRadar` folder are updated by the GitHub Actions. There's no need to review them in detail.

### CODE REVIEW

File: .github/workflows/tech_radar.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        id: repository_list
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: >
+          echo repository_list_json=$(
+          gh repo list ${{ github.repository_owner }}
+          --limit 1000
+          --source
+          --no-archived
+          --visibility public
```
Comment (defagos) [inline .github/workflows/tech_radar.yml]: We probably want to have private repositories as well.

**Review by defagos — APPROVED** [review]

**Comment by MGaetan89** [comment]
This is now ready to review. But we should discuss @defagos's question (https://github.com/SRGSSR/guilde-plateformes-propres/pull/11#discussion_r1503280940) in the next meeting before merging this.
Also, the commit https://github.com/SRGSSR/guilde-plateformes-propres/pull/11/commits/cc3c2cb45956fbc2b7e32a146a6095ca520b9137 should be reverted, so the workflow can create PRs when needed.

File: techRadar/used_languages.tmpl [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Used languages
+
+This page lists all the repositories of the SRG SSR GitHub organisation, grouped by primary
+language.
+Some repository may use more than one language, which is not reflected in this document.
```
Comment (defagos) [inline techRadar/used_languages.tmpl]:
```suggestion
Some repositories may use more than one language, which is not reflected in this document.
```

**Review by defagos — APPROVED** [review]

**Comment by MGaetan89** [comment]
Closing this PR, as it is probably not the way to go.

**Pull request closed without merging**